### DISCUSSION TITLE
Open for sponsorships?

### DISCUSSION THREAD

**Discussion by pascualstromsnes** [magic-test/magic-test-laravel#22] [General]
Hello there!

Just came across this amazing package and I would love to sponsor this package, but there seems to be an error in your sponsorship settings.

Amazing work so far! I can really see this package taking off very soon.

**Comment by mateusjatenee** [comment]
Hey Pascual! Thanks so much for reaching out.
GitHub sponsors is currently not supported in Brazil, sadly. 

I really appreciate your kind words and thanks for buying TDD With Laravel! 🙂🚀
